### PR TITLE
Features/adding change log

### PR DESCRIPTION
**Pull request by sumitAgrawal007**
Including new features in the changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,17 +5,14 @@
...
-**ENHANCEMENTS:**
+**FEATURES:**
 
-- Changed number of hosts in main.tf for AWS [\#13](https://github.com/vmware/terraform-provider-vmc/pull/13)
-- Changes to \*.tf and readme in VMC provider for compatibility with terraform v0.12 [\#12](https://github.com/vmware/terraform-provider-vmc/pull/12)
-- Features/iac 12 [\#6](https://github.com/vmware/terraform-provider-vmc/pull/6)
+- resource/vmc_sddc: Support for CRUD operations on SDDCs in VMware Cloud.
+- data_source/vmc_org: Support to read ORGs in VMC.
+- data_source/vmc_connected_accounts: Support to read connected accounts in the given org. 
+- data_source/vmc_customer_subnets: Support to read customer subnets in the given org and account.
```
Comment (shylasrinivas) [inline CHANGELOG.md]: Do we have to mention specific resources and data sources in features section? Also the data sources which provide support for reading data are not a feature IMO as these are used internally to create SDDC..

**Comment by saloneerege** [comment]
Can @rchadalawada confirm if the version is correct ?

**Review by rchadalawada — COMMENTED** [review]
Refresh the changelog to include the 2 new enahncements we made: nsx_reverse_proxy & filtering based on aws account

**Comment by saloneerege** [comment]
> Refresh the changelog to include the 2 new enahncements we made: nsx_reverse_proxy & filtering based on aws account

Done

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,14 +5,20 @@
...
 
-**FEATURES:**
+FEATURES:
 
-- resource/vmc_sddc: Support for CRUD operations on SDDCs in VMware Cloud.
-- data_source/vmc_org: Support to read ORGs in VMC.
-- data_source/vmc_connected_accounts: Support to read connected accounts in the given org. 
-- data_source/vmc_customer_subnets: Support to read customer subnets in the given org and account.
+* resource/vmc_sddc: Support for CRUD operations on SDDCs in VMware Cloud.
+* data_source/vmc_org: Support to read ORGs in VMC.
```
Comment (shylasrinivas) [inline CHANGELOG.md]: what does ORGs stand for? should we use organization instead?

**Review by shylasrinivas — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (saloneerege) [reply CHANGELOG.md]: I see this pattern of adding the data source related information in the Changelog for other providers like AWS. I think we can keep it.

File: CHANGELOG.md (hunk 2 above)
Comment (saloneerege) [reply CHANGELOG.md]: Done

**Review by rchadalawada — APPROVED** [review]

**Pull request merged**